mipengine/controller/config.toml

```diff
@@ -6,7 +6,11 @@ cdes_metadata_path = "$CDES_METADATA_PATH"
 deployment_type="$DEPLOYMENT_TYPE"
 
 node_registry_update_interval="$NODE_REGISTRY_UPDATE_INTERVAL"
-nodes_cleanup_interval=2
+
+[cleanup]
+contextids_cleanup_file="/tmp/contextids_cleanup.toml"
```

Comment:
Is it safe to write this file in the /tmp/ folder? Won't it be cleaned when the container closes, even though we have it volumed?

I am really not sure how this will work, if you know it won't be cleaned, I have no problem leaving it as is.

Reply:
Changed to `/opt/cleanup` just to be sure. It's only container internal file anyway.